linode_api4/objects/base.py

```diff
@@ -210,9 +210,29 @@ def save(self, force=True) -> bool:
         if not force and not self._changed:
             return False
 
-        resp = self._client.put(
-            type(self).api_endpoint, model=self, data=self._serialize()
-        )
+        if not self._populated:
+            result = {
+                a: getattr(self, a)
```

Comment:
`getattr` -> `object.__getattribute__`

Reply:
For context, this is how we bypass the overridden `__getattribute__(...)` in `Base`: https://docs.python.org/3/reference/datamodel.html#object.__getattribute__

Reply:
Fixed!